media/css/firefox/new/fx-lifestyle/conformity-not-default.less

```diff
@@ -14,6 +15,10 @@
         p {
             .font-size(22px);
         }
+
+        .copy {
+            width: 500px;
```

Comment:
Had to adjust the width here to accommodate slightly more copy, given that the images are all fixed height :/